cloud_edition/flexibility_mode/docs/composer_settings.rst

```diff
@@ -0,0 +1,58 @@
+Composer settings
+=================
+
+
+
+| During the install process we used our own repository for the EE distribution. But you don't have any access to this repository after the install process.
+| We setup your own distribution repository in the composer file, so you can change it if you want.
+
+
+| To be able to pull from distribution, you must use a known ssh keypair from partner portal.
+| You can check your access rights to the distribution repository with this command:
+
+.. code-block:: bash
+
+    $ ssh git@distribution.akeneo.com -p 443
+    akeneo@my-instance-staging:~$ ssh git@distribution.akeneo.com -p 443
+    PTY allocation request failed on channel 0
+    hello papo-full_name, this is git@distribution running gitolite3 3.6-1~bpo70+1 (Debian) on git 1.7.10.4
+
+    R  	ElasticSearchBundle
+    R 	    InnerVariationBundle
+    R 	    LdapAuthenticationBundle
+    R 	    pim-enterprise-dev-partner
+
+
+| After that, you can set up your composer.json like that
+
+.. code-block:: json
+
```

Comment:
sphinx lint error. Json fail because of a missing "{"
Don't forget the blank line between `.. code-block:: json` and `{`

```
.. code-block:: json

    {
        "type": "vcs",
        "url": "ssh://git@distribution.akeneo.com:443/pim-enterprise-dev-<who_i_am>.git",
        "branch": "master"
    }
```